packages/plugin-dev/package.json

```diff
@@ -1,52 +1,55 @@
-{
-  "name": "@theia/plugin-dev",
-  "version": "0.8.0",
-  "description": "Theia - Plugin Development Extension",
-  "main": "lib/common/index.js",
-  "typings": "lib/common/index.d.ts",
-  "dependencies": {
-    "@theia/core": "^0.8.0",
-    "@theia/plugin-ext": "^0.8.0",
-    "@theia/preferences": "^0.8.0",
-    "ps-tree": "1.1.0"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "theiaExtensions": [
-    {
-      "backend": "lib/node/plugin-dev-backend-module",
-      "backendElectron": "lib/node-electron/plugin-dev-electron-backend-module",
-      "frontend": "lib/browser/plugin-dev-frontend-module"
-    }
-  ],
-  "keywords": [
-    "theia-extension"
-  ],
-  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/theia-ide/theia.git"
-  },
-  "bugs": {
-    "url": "https://github.com/theia-ide/theia/issues"
-  },
-  "homepage": "https://github.com/theia-ide/theia",
-  "files": [
-    "lib",
-    "src"
-  ],
-  "scripts": {
-    "prepare": "yarn run clean && yarn run build",
-    "clean": "theiaext clean",
-    "build": "theiaext build",
-    "watch": "theiaext watch",
-    "test": "theiaext test"
-  },
-  "devDependencies": {
-    "@theia/ext-scripts": "^0.8.0"
-  },
-  "nyc": {
-    "extends": "../../configs/nyc.json"
-  }
-}
+{
+  "name": "@theia/plugin-dev",
+  "version": "0.8.0",
+  "description": "Theia - Plugin Development Extension",
+  "main": "lib/common/index.js",
+  "typings": "lib/common/index.d.ts",
+  "dependencies": {
+    "@theia/core": "^0.8.0",
+    "@theia/workspace": "^0.8.0",
+    "@theia/filesystem": "^0.8.0",
+    "@theia/plugin-ext": "^0.8.0",
+    "@theia/debug": "^0.8.0",

```

Comment:
Why is `@theia/debug` needed?

Reply:
It is used here: https://github.com/theia-ide/theia/blob/d45d6337faad6199042fc593e44fe4080b5d7dd7/packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts#L28